streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -275,6 +277,18 @@
     @SuppressWarnings("WeakerAccess")
     public static final String UPGRADE_FROM_23 = "2.3";
 
+    /**
+     * Config value for parameter {@link #UPGRADE_FROM_CONFIG "upgrade.from"} for upgrading an application from version {@code 2.4.x}.
+     */
+    @SuppressWarnings("WeakerAccess")
+    public static final String UPGRADE_FROM_24 = "2.4";
+
+    /**
+     * Config value for parameter {@link #UPGRADE_FROM_CONFIG "upgrade.from"} for upgrading an application from version {@code 2.5.x}.
+     */
+    @SuppressWarnings("WeakerAccess")
+    public static final String UPGRADE_FROM_25 = "2.5";
+
```

Comment:
This is not really relevant for this PR, but we need to add it for KIP-447 eventually, thus I just include it in this PR.